src/CloudNative.CloudEvents.AspNetCore/CloudEventInputFormatter.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Cloud Native Foundation. 
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+namespace CloudNative.CloudEvents
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc.Formatters;
+    using Microsoft.Net.Http.Headers;
+
+    public class CloudEventInputFormatter : TextInputFormatter
+    {
+        public CloudEventInputFormatter()
+        {
+            SupportedMediaTypes.Add(MediaTypeHeaderValue.Parse("application/json"));
+            SupportedMediaTypes.Add(MediaTypeHeaderValue.Parse("application/cloudevents"));
```

Comment:
Yeah, with the name change to CloudEventJsonInputFormatter, it's reasonable to only support `application/cloudevents+json`. I think the general idea is that more formats for cloudevents could be added (to the other request extensions at least), but since all of them only supports Json, we can too.